Sources/SocksCore/SocketOptions+Deprecated.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  SocketOptions+Deprecated.swift
+//  Socks
+//
+//  Created by Andreas Ley on 20.10.16.
+//
+//
+
+#if os(Linux)
+    import Glibc
+    private let s_socket = Glibc.socket
+    private let s_close = Glibc.close
+#else
+    import Darwin
+    private let s_socket = Darwin.socket
+    private let s_close = Darwin.close
+#endif
+
+extension RawSocket {
+    
+    /// Returns the current error code of the socket (0 if no error)
+    @available(*, deprecated, message: "use getErrorCode() instead")
+    public var errorCode: Int32 {
```

Comment:
We should be able to tag this as a minor release since the public API appears to be intact, right?